src/zarate.jl

```diff
@@ -121,6 +124,13 @@ function ordering_problem(layout::Zarate, graph, layer2nodes)
                 end
             end
         end
+
+        # add optional ordering
+        for (k, v) in opt_node_order #convention k > v
+            if (k in nodes) && (v in nodes)# ordering applies only if they belong to the same layer
```

Comment:
```suggestion
        for (k, v) in opt_node_order  # convention k > v
            if (k in nodes) && (v in nodes)  # ordering applies only if they belong to the same layer
```
Minor style quibble:
BlueStyle likes 2 spaces before and 1 space after `#`